CodeStarterCamp_Week3/step1.swift

```diff
@@ -0,0 +1,70 @@
+class Person {
+    var name: String
+    var money: Int
+    var coffeeList: [Coffee]
+    
+    init(name: String, money: Int) {
+        self.name = name
+        self.money = money
+        self.coffeeList = []
+    }
```

Comment:
```suggestion
    var coffeeList: [Coffee] = []
    
    init(name: String, money: Int) {
        self.name = name
        self.money = money
    }
```

초기화할 때 따로 전달을 받지 않고 빈 배열을 할당한다면 프로퍼티 기본 값을 사용하는 것이 더 좋을 것 같습니다.